diesel_infer_schema/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "diesel_infer_schema"
+version = "0.1.0"
```

Comment:
Memo to @diesel-rs/core: Need to publish this crate.

FYI, its API surface is quite small. Just two function with the signature `(database_url: &str, table_name: &str) -> Tokens`.